test/testscript.py

```diff
@@ -0,0 +1,6 @@
+import pymongo
+import sys
+print(sys.argv)
```

Comment:
Please remove this line - seems like it is for debugging.

Reply:
Done.